arches_for_science/migrations/0005_deploy_triggers_for_primary_descriptor.py

```diff
@@ -58,7 +58,6 @@ class Migration(migrations.Migration):
                     operation="INSERT OR UPDATE OR DELETE",
                     pgid="pgtrigger_calculate_multicard_primary_descriptor_single_26494",
                     table="tiles",
-                    timing="DEFERRABLE INITIALLY DEFERRED",
```

Comment:
We should just be careful to remake migrations instead of trying to edit the files; luckily CI caught the drift between the trigger defs and the calculated state.